modules/exploits/unix/webapp/spip_rce_form.rb

```diff
@@ -0,0 +1,124 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'SPIP form PHP Injection',
+        'Description' => %q{
+          This module exploits a PHP code injection in SPIP. The vulnerability exists in the
+          oubli parameter and allows an unauthenticated user to execute arbitrary commands
+          with web user privileges. Branches 3.2, 4.0, 4.1 and 4.2 are concerned. Vulnerable versions
+          are <3.2.18, <4.0.10, <4.1.18 and <4.2.1.
+        },
+        'Author' => [
+          'anonymous', # Initial discovery
+          'Laluka',         # PoC
+          'Julien Voisin'   # MSF module
+        ],
+        'License' => MSF_LICENSE,
+        'References' => [
```

Comment:
@todb-r7 This PR is missing a CVE, but it has been patched, would you happen to have any guidance here? Thanks.

Reply:
Hey @jvoisin, could you confirm if this exploit corresponds to this CVE: https://nvd.nist.gov/vuln/detail/CVE-2023-27372?

Reply:
Yes it is.
I also added it to the module :)